### PR TITLE
record unknown answer label for out of bounds lookups

### LINKED ISSUE #2104: not receiving comet hunters classification export
I'm not getting the Comet Hunters classification data export. I do get Planet Four: Terrains. I waited for 18 hours with no email. I tried again today and it's on the 5 hour mark and no export has been received. I think something's wrong/breaking for Comet Hunters. Can someone take a look?

### PR DESCRIPTION
**Pull request by camallen**
Closes #2104 - some annotations have out of bounds index lookups for the workflows answers array, this will mark them as invalid but still record the data in the export for the researcher.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: lib/formatter/csv/annotation_for_csv.rb [hunk 1]
```diff
@@ -145,9 +145,10 @@ def answer_label
       def answer_labels
         Array.wrap(@current["value"]).map do |answer_idx|
           begin
-            answer_string = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]['label']
+            task_answer = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]
```
Comment (marten) [inline lib/formatter/csv/annotation_for_csv.rb]: Could we use `Hash#dig` instead here to fetch nested values (works across hashes with arrays with hashes just fine)? That way we don't have to capture all `NoMethodError`s. But we'd get a `nil` back, and we wouldn't be able to distinguish between a label that is configured to be `nil` and one that is not configured. Don't think that's an issue, but sanity check me here.

Comment (zwolf) [inline lib/formatter/csv/annotation_for_csv.rb]: Agreed that there needs to be something in the invalid label's place for researchers.
